var/spack/repos/builtin/packages/biobb-common/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class BiobbCommon(PythonPackage):
```

Comment:
Is this a command-line tool or a Python API? See https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pythonpackage-vs-packages-that-use-python

Reply:
It is both.
Every biobb (this is just the first from a long list) is prepared to run both in command line and from python.
They are actually also prepared to run in docker, singularity, cwl and galaxy.

Reply:
We generally prefer to prefix Python libraries with `py-` unless they are predominantly used from the command line. For example, mercurial is technically a Python library, but that's not important to most users, so the package is called `mercurial`. As a counterexample, numpy as command-line tools, but most people don't know or care about that, they just want to import it in their Python code, so the package is called `py-numpy`.

Reply:
done